src/lib/filter-utils.ts

```
@@ -0,0 +1,207 @@
+
+    case 'after':
+      return filterValue ? new Date(pageValue) > new Date(filterValue) : false;
+
+    case 'on_or_before':
+      return filterValue ? new Date(pageValue) <= new Date(filterValue) : false;
+
+    case 'on_or_after':
+      return filterValue ? new Date(pageValue) >= new Date(filterValue) : false;
+
+    case 'past_week':
+      const weekAgo = new Date();
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)